tests/jobs/test_db.py

```diff
@@ -1,12 +1,12 @@
-from virtool.jobs.db import processor
+from virtool_core.models.job import Job
 
+from virtool.jobs.db import fetch_complete_job
 
-async def test_processor(snapshot, mongo, fake2, static_time, test_job):
-    """
-    Test that the dispatch processor properly formats a raw job document into a
-    dispatchable format.
-
-    """
+async def test_fetch_complete_job(snapshot, mongo, fake2, static_time, test_job):
     user = await fake2.users.create()
     test_job["user"] = {"id": user.id}
-    assert await processor(mongo, test_job) == snapshot
+
+    result = await fetch_complete_job(mongo, test_job)
+
+    assert isinstance(result, Job)
+    snapshot.assert_match(result)
```

Comment:
Simplify this. It is obvious from the snapshot what the type of the return value is.
```suggestion
    assert await fetch_complete_job(mongo, test_job) == snapshot
```